src/index.html

```diff
@@ -7,6 +7,12 @@
     <title>Capstone Project</title>
 </head>
 <body>
-    
+    <header>
+        <nav class="navigation">
+            <div class="logo-container"><img class="logo" src="assets/images/logo.png" alt="meme logo"></div>
```

Comment:
- [ ] Image is nice. But if has no background it will be more nice.

![Screenshot from 2021-08-09 21-30-28](https://user-images.githubusercontent.com/38635158/128756106-1b87567b-d9cd-44ea-8ae5-8cc464e9078d.png)